hpx/parallel/algorithms/exclusive_scan.hpp

```diff
@@ -22,6 +22,7 @@
 #include <hpx/parallel/util/loop.hpp>
 #include <hpx/parallel/util/partitioner.hpp>
 #include <hpx/parallel/util/scan_partitioner.hpp>
+#include <hpx/parallel/util/projection_identity.hpp>
```

Comment:
Please try to keep the headers sorted alphabetically (inside each of the blocks). Clang-format will do that for you.